roles/0-init/tasks/validate_vars.yml

```
@@ -63,7 +63,7 @@
 #
 # 2020-11-04: Fix validation of 5 [now 4] core dependencies, for ./runrole etc
 
-- name: Set vars_checklist for 44 + 44 + 40 vars ("XYZ_install" + "XYZ_enabled" + "XYZ_installed") to be checked
+- name: Set vars_checklist for 45 + 45 + 41 vars ("XYZ_install" + "XYZ_enabled" + "XYZ_installed") to be checked
```

@cwivagg can you update this 1 line?

```suggestion
- name: Set vars_checklist for 46 + 46 + 42 vars ("XYZ_install" + "XYZ_enabled" + "XYZ_installed") to be checked
```

Thanks if possible as another PR (#3302) just landed right in front of us :smile:

- https://github.com/iiab/iiab/pull/3302/files#diff-37a60cf03710ef308e5b429635f5eb815ab05b23850b94330c0c873036237381